en/docs/fragments/remove-users.md

```diff
@@ -0,0 +1,8 @@
+1.  On the **Main** tab in the management console, click **List** under
```

Comment:
1.  In the **Main** menu under the **Identity** section, click **List** under  **Users and Roles**